public/components/common/side_nav.tsx

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
I think this file was taken away in Nav/OSD-Nav change.
This might be mistaken merge.

If this is required somehow we need to discuss alternative.

Reply:
Will remove